cfgov/v1/management/commands/delete_page_cache.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management.base import BaseCommand
+
+from wagtail.contrib.frontend_cache.utils import purge_urls_from_cache
+
+
+class Command(BaseCommand):
+    help = "Delete the cache of pages by full URLs"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--url",
+            required=True,
+            nargs="+",
+            help=(
+                "The full URL for the page cache to delete "
+                "(can specify multiple)"
+            )
+        )
+
+    def handle(self, *args, **options):
+        purge_urls_from_cache(options["url"])
```

Comment:
This is the same as it previously was, but I don't think this will actually work.

`purge_urls_from_cache` [just issues a `purge_batch` to the backend](https://github.com/wagtail/wagtail/blob/main/wagtail/contrib/frontend_cache/utils.py#L104), which will end up with an invalidation. 

I think I'm going to do a bit of refactoring here.